packages/vats/tools/fake-bridge.js

```diff
@@ -300,46 +300,74 @@ export const fakeLocalChainBridgeQueryHandler = message => {
 
 /**
  * @param {import('@agoric/zone').Zone} zone
- * @param {(obj: object) => void} [onToBridge]
+ * @param {(obj: object, result: unknown) => void} [onToBridge] Log message and
+ *   result
  * @param {(index: number) => string} makeAddressFn
  * @returns {ScopedBridgeManager<'vlocalchain'>}
  */
 export const makeFakeLocalchainBridge = (
   zone,
-  onToBridge = () => {},
+  onToBridge = (_obj, _result) => {},
   makeAddressFn = index => `${LOCALCHAIN_DEFAULT_ADDRESS}${index || ''}`,
 ) => {
   /** @type {Remote<BridgeHandler>} */
   let hndlr;
   let lcaExecuteTxSequence = 0;
+  // Removed lastMsgTransferSequence state
   let accountsCreated = 0;
+  // Removed state map
+
   return zone.exo('Fake Localchain Bridge Manager', undefined, {
     getBridgeId: () => 'vlocalchain',
+    // Removed getLastMsgTransferSequence method
```

Comment:
More unclear comments.  Maybe they were products of intermediate aider refactorings that were reverted before pushing?